assets/src/web-stories-block/block/block-types/latest-stories/edit.js

```diff
@@ -122,27 +114,20 @@ const LatestStoriesEdit = ({ attributes, setAttributes }) => {
 
   return (
     <>
-      <StoriesBlockControls
-        blockType={blockType}
-        viewType={viewType}
-        setAttributes={setAttributes}
-      />
```

Comment:
Why is this being removed?

Reply:
Because it was moved to `assets/src/web-stories-block/block/edit.js` to avoid duplication